stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -1280,7 +1281,9 @@ function shell_completions(string, pos)
         # Also try looking into the env path if the user wants to complete the first argument
         use_envpath = !ignore_last_word && length(args.args) < 2
 
-        return complete_path(prefix, pos, use_envpath=use_envpath, shell_escape=true)
+        r = thisind(prefix, pos - length(prefix)):pos # TODO: this is a bad approximation of where the start of this argument was
```

Comment:
seems like there was supposed to be some sort of `last_parse` feature (4467885a5215e767057add4828ebecb7f47ae7bb and 90d0ff5d64efd1aa9aafbb7b22fa68dba09dccc0) but I am not entirely sure how it is supposed to work